src/ClashOfClans/Models/WarClanBase.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace ClashOfClans.Models
+﻿using System;
+
+namespace ClashOfClans.Models
 {
-    public class ClanWarLeagueWarClan : Identity
+    public class WarClanBase : Identity
```

Comment:
ClanWarLeagueWar is the official name in the Supercell API, I'm not going to change the class name to something else because it makes future development more difficult because the names do not match to official API names.